includes/wikia/DefaultSettings.php

```diff
@@ -1817,13 +1817,6 @@
  */
 $wgShortArticlePathWikis = [
```

Comment:
can we get rid of this variable and in `WikiFactory::cityUrlToArticlePath` simply check for the `CORPORATE_WIKI_ID`?